### ISSUE TITLE
Found blocks in Bitcoin testnet with wrong coinbase address

### ISSUE THREAD

**Issue by pbipbi** [zone117x/node-stratum-pool#157]
I set "address" to my, but always found blocks in testnet (several times) with this coinbase "mqGucXZkq2Fp216qhUViqPXx5LVPJNEKf9". I cant understand where bitcoind/stratum get it?

**Comment by HashUnlimited** [comment]
That's because you are using a P2SH address for your "own" and the pool is decoding it without error but wrong. This behaviour is fixed in the dev branch.

**Comment by pbipbi** [comment]
Why this part of code not included to master branch?

**Comment by HashUnlimited** [comment]
indeed, nothing has been reported faulty. @zone117x what is your opinion?
